File: 14-celsius-to-fahrenheit/main.cpp
/*
  Ernest Izdebski, 2022-09-14

  A program that receives the Celsius temperature from the keyboard and
  converts the Celsius temperatures to Fahrenheit temperatures.
*/

#include <iostream>
using namespace std;

int main() {
  // define & populate variable
  cout << "Input degrees in Celsius: ";
  double degrees_c;
  cin >> degrees_c;

  // display output
  cout << degrees_c << "°C is " << (9*degrees_c / 5) + 32 << "°F." << endl; 
}

File: 14-celsius-to-fahrenheit/src/main.rs
mod ernie;

fn c_to_f(deg_c: f32) -> f32 {
    deg_c * 9.0 / 5.0 + 32.0
}

fn main() {
    let deg_c: f32 = ernie::input("enter degrees in celsius: ");
    println!("{}°C is equal to {}°F", deg_c, c_to_f(deg_c));
}
